packages/typespec-test/test/openai_non_branded/generated/typespec-ts/package.json

```diff
@@ -0,0 +1,119 @@
+{
+  "name": "@msinternal/openai-non-branded",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
```

Comment:
Can we also make this configurable?

Reply:
It is better no to have the keyword of Microsoft either.

Reply:
updated